src/DacpacTool/DacpacTool.csproj

```diff
@@ -12,6 +12,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="ErikEJ.DacFX.SqlServer.Rules" Version="1.0.0-preview3" />
```

Comment:
Are these actually needed in the csproj file? I don't think I saw any references to them, but let me know if I'm mistaken

Reply:
They are needed in order to add the additional analyzers .dlls as mentioned in the PR description - I guess that is "runtime" only?

Reply:
@jmezach @jeffrosenberg Should I change anything?

Reply:
If we did an explicit copy, then we probably wouldn't need the `<PackageReference>` at all. I don't have a big problem with using the `<PackageReference>` to copy the files, I just didn't make the connection of what it was doing. At the minimum, I'd probably add a comment clarifying why these are in the project file.

Reply:
How about Includeassets="runtime" ?